Addition of Engine Developer Guide topic and structure stubs (#2162)
o3de/o3de.org: content/docs/engine-dev/gems/_index.md

---
linkTitle: Gems
title: Open 3D Engine Core Gem Developer Documentation
description: Documentation for developers extending or contributing to the Gems bundled as part of Open 3D Engine.
weight: 300
---

{{< note >}}

> This section of the documentation is for developers working on Gems distributed with O3DE, _not_ working on new Gems to add for their project. If you're a user looking to extend the engine through new functionality rather than modifying existing behavior, read the [User Guide - Gem Programming](/docs/user-guide/programming/gems) topic.

Reviewer: This is a contradiction to both the dev guide intro, where it was stated that in this section they'd learn how to "create
your own Gem as an extension", and the paragraph that follows, where it includes "those looking to contribute [Gems]" in the target audience. I think it would be useful to cover creating a Gem in both the dev guide and the user guide. It could be covered in this section, under a separate topic on "Creating Gems", or included in the overall architecture content.
```suggestion
This section of the documentation is for developers who wish to modify Gems that are distributed with O3DE.
```
